services/web/package.json

```diff
@@ -2,26 +2,24 @@
   "name": "bedrock-web",
   "version": "0.0.1",
   "license": "MIT",
+  "type": "module",
   "engines": {
     "node": ">=20"
   },
   "scripts": {
-    "build": "NODE_ENV=production webpack",
-    "start": "node ./serve/dev",
+    "build": "vite build",
+    "start": "vite --port 2200",
```

Comment:
You can just specify that in the config and do just run `vite` here, have a look here: https://github.com/bedrockio/pages/blob/master/web/vite.config.js

Reply:
also here is my template:

https://github.com/rekallai/all-in-all/blob/main/services/landing/src/index.html